cmake/DaemonFlags.cmake

```diff
@@ -238,7 +238,9 @@ else()
 	endif()
 
 	# Optimizations.
+	if (NACL AND USE_NACL_SAIGO)
```

Comment:
What's this change?

Reply:
Something I committed by mistake when I tested:

- https://github.com/DaemonEngine/Daemon/issues/1312

Fixed.